pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Related Issue
+- Link for the issue
+- Description of the issue
+
+# Proposed Changes
+- Change 1
+- Change 2
+- Change 3
+
+# Additional Info
+- Additional content or any information
+
+# Checklist
+- [ ] Have you tested your code?
+- [ ] Does it follow style guides?
+- [ ] Does it also include translations?
+- [ ] Does it include any updates in the documentation?
```

Comment:
This should not be a strict requirement right now, but something that contributors can check if they do update the README
```suggestion
- [ ] Updated `README.md`
```